sihl_core/lib/http.ml

```diff
@@ -123,7 +193,17 @@ module Response = struct
     in
     let code = Cohttp.Code.status_of_code resp.status in
     let body = resp.body |> Option.value ~default:Msg.ok_string in
-    Opium.Std.respond ~headers ~code (`String body)
+    let co_resp = Opium.Std.respond ~headers ~code (`String body) in
+    match resp.session with
+    | Nothing -> co_resp
+    | SetSession token ->
+        Opium.Std.Cookie.set ~http_only:true ~secure:false ~key:"session_id"
+          ~data:token co_resp
+    | StopSession ->
```

Comment:
StopSession sound weird, why not DeleteSession or EndSession?

Reply:
Fixed.